relayer/go.mod

```diff
@@ -196,3 +197,5 @@ require (
 )
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+replace github.com/smartcontractkit/chainlink-starknet/ops => ../ops
```

Comment:
Same thing here, we should avoid replaces in `go.mod` to avoid accidentally building unwanted code/versions. Add a `go.work` and add those replace statements in, but comment them out to be used only during development.

Reply:
Also add the `go.work` to `.gitignore` once you committed it